examples/load-data-via-graph-construction.ipynb

```diff
@@ -0,0 +1,717 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "X5ttqidnWPTC"
+   },
+   "source": [
+    "## Setup\n",
+    "\n",
+    "We need an environment where Neo4j and GDS are available, for example AuraDS (which comes with GDS preinstalled) or Neo4j Desktop. \n",
+    "\n",
+    "Once the credentials to this environment are available, we can install the `graphdatascience` package and import the client class."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "izJD_HlcAdQ0"
+   },
+   "outputs": [],
+   "source": [
+    "!pip install graphdatascience==1.3"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from graphdatascience import GraphDataScience"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When using a local Neo4j setup, the default connection URI is `bolt://localhost:7687`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Replace with the actual connection URI and credentials\n",
+    "NEO4J_CONNECTION_URI = \"bolt://localhost:7687\"\n",
+    "NEO4J_USERNAME = \"neo4j\"\n",
+    "NEO4J_PASSWORD = \"\"\n",
+    "\n",
+    "gds = GraphDataScience(NEO4J_CONNECTION_URI, auth=(NEO4J_USERNAME, NEO4J_PASSWORD))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When using AuraDS, the connection URI is slightly different as it uses the `neo4j+s` protocol. The client should also include the `aura_ds=True` flag to enable AuraDS-recommended settings.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Replace with the actual connection URI and credentials\n",
+    "NEO4J_CONNECTION_URI = \"neo4j+s://xxxxxxxx.databases.neo4j.io\"\n",
+    "NEO4J_USERNAME = \"neo4j\"\n",
+    "NEO4J_PASSWORD = \"\"\n",
+    "\n",
+    "gds = GraphDataScience(NEO4J_CONNECTION_URI, auth=(NEO4J_USERNAME, NEO4J_PASSWORD), aura_ds=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "zOK08QpoWFKv"
+   },
+   "source": [
+    "We also import `pandas` to create a Pandas `DataFrame` from the original data source."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {
+    "id": "-ikcvI-cSDoK"
+   },
+   "outputs": [],
+   "source": [
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "oDE2LqTXAmy8"
+   },
+   "source": [
+    "## Load the Cora dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {
+    "id": "tOZW-I2FBLm_"
+   },
+   "outputs": [],
+   "source": [
+    "# TODO: use URLs within the client repo when the notebook is added there\n",
```

Comment:
FYI `gds.graph.load_cora` is merged now. But I don't think that should affect this notebook. However, I don't think we will necessarily have the content files raw in this repo so this TODO probably isn't relevant anymore.

Reply:
Yeah I think it's fine, but maybe we still should have a TODO actually regarding putting the data in a more suitable/permanent place. Either is fine I think :)